docs/source/guides/rsthelp.rst

```diff
@@ -0,0 +1,476 @@
+The text and content may be misaligned so you just end desyncing up your documentation for the reader.
+
+Like now. The previous text has nothing to do with the cute cat. Now that we're talking about the cat though.
+Isn't that cat cute? Imagine sleeping that good. Maybe it's napping.
+I wonder if the ratio of cute sleeper are higher for cats than for humans.
+
+Anyways.
+
+.. important:: You can align with ``:align: <parameter>``, available parameters are ``right``, ``left``, ``center``, ``top``, ``middle`` or ``bottom``.
+
+Sources: `https://docutils.sourceforge.io/docs/ref/rst/directives.html#images <https://docutils.sourceforge.io/docs/ref/rst/directives.html#images>`_, `https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#images-and-figures <https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#images-and-figures>`_
+
+Admonitions:
+-----
+
+These are warnings or hazard statements.
+
+Examples:
+#####
+
+.. attention:: This is done ``.. attention:: YOUR_TEXT HERE``
+.. danger:: This is done ``.. danger:: YOUR_TEXT HERE``
+.. error:: This is done ``.. error:: YOUR_TEXT HERE``
+.. caution:: This is done ``.. caution:: YOUR_TEXT HERE``
+.. warning:: This is done ``.. warning:: YOUR_TEXT HERE``
+.. note:: This is done ``.. note:: YOUR_TEXT HERE``
+.. important:: This is done ``.. important:: YOUR_TEXT HERE```
+.. hint:: This is done ``.. hint:: YOUR_TEXT HERE``
+.. tip:: This is done ``.. tip:: YOUR_TEXT HERE``
+or topics which are very similar to admonitions.
+
+.. topic:: Topic/Title
+
+    Description
+
+Source: `https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions <https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions>`_
+
+References:
+-----
+
+TODO - REALLY great for tables. Basically text links (i haven't done much testing).
+
+Example:
+::
+
+    .. |title| replace::
+        This is realllllly useful for tables. This is reference usage.
+
+This is used right now. Check below for usage in example tables.
+
+.. |title| replace::
+        This is realllllly useful for tables. This is reference usage.
+
+Tables:
+-----
+
+TODO - Tables. Before learning tables. Learn References.
+
+Types: 
+#####
+
+Simple Simple:
+::
+
+    ====== ====== ============================
+     hi     elo      IM A Table
+    ====== ====== ============================
+
+====== ====== ============================
+ hi     elo      IM A Table
+====== ====== ============================
+
+Simple: 
+:: 
+
+    ====== ====== ============================
+    hi     elo     Titles
+    ====== ====== ============================
+    hi     elo      |title|
+    ====== ====== ============================
+
+====== ====== ============================
+ hi     elo     Titles
+====== ====== ============================
+  hi     elo      |title|
+====== ====== ============================
+
+Complex Complex:
+
+TODO
+
+Complex:
+
+TODO
+
+Complex Easiest:
+
+TODO
```

Comment:
More `TODO`s ^^

Reply:
Just noting down the organization for future PR or commits.